synapse/models/inet.py

```diff
@@ -825,62 +909,235 @@ def getModelDefs(self):
 
                     )),
 
-                    ('inet:whois:rar', {}, ()),
-
-                    ('inet:whois:reg', {}, ()),
+                    ('inet:web:action', {}, (
+                        ('act', ('str', {'lower': True}), {
+                            'doc': 'The action performed by the account.'
+                        }),
+                        ('acct', ('inet:web:acct', {}), {
+                            'ro': 1,
+                            'doc': 'The web account associated with the action.'
+                        }),
+                        ('acct:site', ('inet:fqdn', {}), {
+                            'ro': 1,
+                            'doc': 'The site or service associated with the account.'
+                        }),
+                        ('acct:user', ('inet:user', {}), {
+                            'ro': 1,
+                            'doc': 'The unique identifier for the account.'
+                        }),
+                        # FIXME missing json
+                        #('info', ('json', {}), {
+                        #    'doc': 'Any other data associated with the action.'
+                        #}),
+                        ('time', ('time', {}), {
+                            'doc': 'The date and time the account performed the action.'
+                        }),
+                        ('ipv4', ('inet:ipv4', {}), {
```

Comment:
this feels like it rolls up into `inet:client`

Reply:
agreed